src/web/app/pages-help/instructor-help-page/instructor-help-questions-section/instructor-help-questions-section.component.html

```diff
@@ -538,7 +538,11 @@ <h2 class="color-blue" *ngIf="showQuestion.length === 0 || showQuestion.includes
           <p>
             To respondents, a rubric question will appear as a table that looks similar to the example below. Respondents can choose one answer per row.
           </p>
-          <p>EXAMPLE BOX</p>
+          <tm-example-box>
+            <tm-question-submission-form
+                [formModel]="exampleResponderRubricSubmissionFormModel"
```

Comment:
If I'm not wrong you can add `[isDisabled]="true"`

Reply:
done.